sdk/neonpostgres/Azure.ResourceManager.NeonPostgres/api/Azure.ResourceManager.NeonPostgres.net8.0.cs

```diff
@@ -271,6 +307,25 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.NeonPostgres.Models.DefaultEndpointSettings>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct EndpointStatus : System.IEquatable<Azure.ResourceManager.NeonPostgres.Models.EndpointStatus>
```

Comment:
this name is too generic, could we add something to it so that it could be more specific?